<ui version="4.0" >
 <class>FileProperties</class>
 <widget class="QWidget" name="FileProperties" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>File Properties</string>
  </property>
  <widget class="QWidget" name="horizontalLayout" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>520</y>
     <width>401</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="spacing" >
     <number>6</number>
    </property>
    <property name="leftMargin" >
     <number>0</number>
    </property>
    <property name="topMargin" >
     <number>0</number>
    </property>
    <property name="rightMargin" >
     <number>0</number>
    </property>
    <property name="bottomMargin" >
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="ReloadButton" >
      <property name="text" >
       <string>&amp;Reload</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="ButtonBox" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons" >
       <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="PropertiesTabWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>501</height>
    </rect>
   </property>
   <property name="currentIndex" >
    <number>0</number>
   </property>
   <widget class="QWidget" name="FilePropertiesTab" >
    <attribute name="title" >
     <string>File Properties</string>
    </attribute>
    <widget class="QWidget" name="verticalLayout" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>381</width>
       <height>461</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="FileInformationLabel" >
        <property name="text" >
         <string>&lt;u>&lt;b>File Information&lt;/b>&lt;/u></string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <property name="horizontalSpacing" >
         <number>6</number>
        </property>
        <property name="verticalSpacing" >
         <number>6</number>
        </property>
        <item row="1" column="0" >
         <widget class="QLabel" name="PathLabel" >
          <property name="text" >
           <string>Path:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLabel" name="Name" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="SizeLabel" >
          <property name="text" >
           <string>Size:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="CreatedLabel" >
          <property name="text" >
           <string>Created:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLabel" name="Path" >
          <property name="text" >
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="NameLabel" >
          <property name="text" >
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" >
         <widget class="QLabel" name="ModifiedLabel" >
          <property name="text" >
           <string>Modified:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QLabel" name="Size" >
          <property name="text" >
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QLabel" name="Created" >
          <property name="text" >
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" >
         <widget class="QLabel" name="Modified" >
          <property name="text" >
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="SoundInformationLabel" >
        <property name="text" >
         <string>&lt;u>&lt;b>Sound Information&lt;/b>&lt;/u></string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <property name="horizontalSpacing" >
         <number>6</number>
        </property>
        <property name="verticalSpacing" >
         <number>6</number>
        </property>
        <item row="0" column="1" >
         <widget class="QLabel" name="Duration" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="DurationLabel" >
          <property name="text" >
           <string>Duration:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="SampleRateLabel" >
          <property name="text" >
           <string>Sample Rate:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLabel" name="SampleRate" >
          <property name="text" >
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="BitPerSampleLabel" >
          <property name="text" >
           <string>Bits Per Sample:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" >
         <widget class="QLabel" name="BitrateLabel" >
          <property name="text" >
           <string>Bitrate:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" >
         <widget class="QLabel" name="CodecLabel" >
          <property name="text" >
           <string>Codec:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" >
         <widget class="QLabel" name="ToolLabel" >
          <property name="text" >
           <string>Tool:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="ChannelsLabel" >
          <property name="text" >
           <string>Channels:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" >
         <widget class="QLabel" name="EncodingLabel" >
          <property name="text" >
           <string>Encoding:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QLabel" name="Channels" >
          <property name="text" >
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QLabel" name="BitsPerSample" >
          <property name="text" >
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" >
         <widget class="QLabel" name="Bitrate" >
          <property name="text" >
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1" >
         <widget class="QLabel" name="Codec" >
          <property name="text" >
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1" >
         <widget class="QLabel" name="Tool" >
          <property name="text" >
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1" >
         <widget class="QLabel" name="Encoding" >
          <property name="text" >
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>PropertiesTabWidget</tabstop>
  <tabstop>ButtonBox</tabstop>
  <tabstop>ReloadButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
